targets/csharp/templates/PlayFabAuthenticationContext.cs.ejs

```diff
@@ -4,15 +4,15 @@ namespace PlayFab
     {
```

Comment:
Repeating what I told you in person here for the sake of logging it.
`PlayFabCommonRequest` now has this field and hence every `_______Request` does too. Since we're trying to now pass the authentication information in the request here instead of the header, we need a property with `{ get; set; }` instead of a field due to JSON serialization/deserialization concerns.